app/controllers/transportMeans/departure/IdentificationController.scala

```diff
@@ -56,27 +59,41 @@ class IdentificationController @Inject() (
     .requireData(lrn)
     .async {
       implicit request =>
+        val viewModel              = viewModelProvider(request.userAnswers, departureIndex)
+        def inlandMode: InlandMode = request.userAnswers.get(InlandModePage).getOrElse(InlandMode("", ""))
+        val para: String = (inlandMode.code, departureIndex) match {
+          case ("3", Index(0)) => viewModel.paragraph1
+          case ("3", _)        => viewModel.paragraph2
+          case _               => ""
+        }
         service.getMeansOfTransportIdentificationTypes(request.userAnswers.get(InlandModePage)).map {
           identificationTypes =>
             val preparedForm = request.userAnswers.get(IdentificationPage(departureIndex)) match {
               case None        => form(identificationTypes)
               case Some(value) => form(identificationTypes).fill(value)
             }
 
-            Ok(view(preparedForm, lrn, identificationTypes, mode, departureIndex))
+            Ok(view(preparedForm, lrn, identificationTypes, mode, departureIndex, viewModel, para))
         }
     }
 
   def onSubmit(lrn: LocalReferenceNumber, mode: Mode, departureIndex: Index): Action[AnyContent] = actions
     .requireData(lrn)
     .async {
       implicit request =>
+        val viewModel              = viewModelProvider(request.userAnswers, departureIndex)
+        def inlandMode: InlandMode = request.userAnswers.get(InlandModePage).getOrElse(InlandMode("", ""))
+        val para: String = (inlandMode.code, departureIndex) match {
+          case ("3", Index(0)) => viewModel.paragraph1
+          case ("3", _)        => viewModel.paragraph2
+          case _               => ""
+        }
```

Comment:
Duplicated logic - all the more reason for this to be defined just once in the view model